index.html

```diff
@@ -303,15 +303,14 @@ <h4>Game streaming</h4>
 transported using RTP or RTCDataChannel.</p>
 </section> 
 <section id="auction">
-   <h4>Low latency Broadcast with Fanout</h4>
-   <p>There are streaming applications that require large scale as well as low latency.
-   Examples include sporting events, church services, webinars and company 'Town Hall' meetings.
-   Live audio, video and data is sent to thousands (or even millions) of recipients.
-   Limited interactivity may be supported, such as allowing authorized participants to ask
-   questions at a company meeting. Both the media sender and receivers may be behind a NAT.
-   P2P relays may be used to improve scalability, potentially using different transport than
-   the original stream.</p>
-   <p class="note">This use case has completed a Call for Consensus (CfC) [[?CFC-Low-Latency]] but has unresolved issues.</p>
+   <h4>Ultra Low latency Broadcast with Fanout</h4>
+   <p>There are streaming applications that require large scale as well as ultra low latency
+      (glass-glass latency less than 300ms).  Examples include auctions. Live audio, video
+      and data is sent to thousands (or even millions) of recipients. Limited interactivity
+      may be supported, such as capturing audio/video from auction bidders.  Both the media
+      sender and receivers may be behind a NAT.  Peer-to-peer relays may be used to improve
+      scalability, with ingestion, distribution and fanout utilizing RTP.
```

Comment:
Since we seem to be having the debate about "but you should consider whether or not datachannels are an equally good solution" at every juncture where we propose extensions that assume the use of RTP, I wonder if it would make sense to add sub-use cases where one is "RTP is used for media transport", and one is "non-RTP protocols are used for media transport in some or all parts of the solution".

That would give us some clean identifiers to point to when saying "THIS extension is applicable to THAT use case".

But I think we should merge this PR and take that up later.

Reply:
This use case is focused on RTP because datachannel-related requirements are already covered in Section 3.1 File Sharing.